styles/components/_button.scss

```diff
@@ -1,4 +1,4 @@
-button {
+%button {
```

Comment:
A wild EXTEND has appeared!

These things are great, makes it so you can copy the styles of another element!

I personally like writing my buttons like this, makes it so we can create lots of variations super easily!

Reply:
Also I use % instead of a class here because the percent symbol is dedicated to extends. It also works with classes and id's but it makes things really clear that this code will ONLY be used as an extend

When you start extending classes all over the place, it's easy to get lost!